DiplomaTests/DiplomaTests/Navigation/LeftMenuNavigation.cs

```diff
@@ -0,0 +1,75 @@
+﻿using DiplomaTests.Pages;
+using DiplomaTests.Pages.AdminPage;
+using DiplomaTests.Pages.HomePage;
+using DiplomaTests.Pages.LeavePage;
+using DiplomaTests.Pages.PerformancePage;
+using DiplomaTests.Pages.PIMPage;
+using DiplomaTests.Pages.RecruitmentPage;
+using DiplomaTests.Utility;
+using OpenQA.Selenium;
+
+namespace DiplomaTests.Navigation
+{
+    public class LeftMenuNavigation : PageBase
+    {
+        private string MenuName = string.Empty;
+
+        private WebElementWrapper MenuLinks => FindElement(By.XPath($"//ul[@class='oxd-main-menu']//li[normalize-space()='{MenuName}']"));
+        private WebElementWrapper NavigationSearch => FindElement(By.XPath($"//nav//input[@placeholder='Search']"));
+
+        public void SeachForNavigationOption(string menuName)
```

Comment:
do you really need menu name in all this menu methods?
how you can handle it in another way:
instead of passing name as method parameter for specific method like ClickPIMMenu, you may:
1. Move all Menu names in constants file 
2. Store 'MenuLinks' as XPath rather then WebElemenent (like private const string menuItem =  //ul[@class='oxd-main-menu']//li[normalize-space()='0'])
3. Create 'generic' method that will take menu name as string parameter and return new WebElement
4. call this generic method in any specific menu method, ex. CLickPIMMenu and pass menu item as parameter to this method